core/server.go

```diff
@@ -58,6 +58,10 @@ func (params *ServerParams) Start() error {
 		}
 	}
 
+	if params.logrusLogger != nil && params.logrusLogger.ReportCaller {
+		ctx = log.WithConfigs(log.SetLogCaller(true)).Onto(ctx)
```

Comment:
I think this would be slightly cleaner inside the section above where you're already dealing with the `logrus` logger:
```go
configs = append(configs, log.SetLogCaller(params.logrusLogger.ReportCaller))
```